docs/external_secrets.md

```diff
@@ -0,0 +1,155 @@
+# External Secrets Operator
+
+ "External Secrets Operator" is a tool that was created by the Kubernetes community to manage external secrets in a Kubernetes cluster. It allows you to fetch and sync external secret values from various external secret management systems into Kubernetes secrets. One of the external secret management systems it can connect to is AWS Secrets Manager. AWS Secrets Manager allows for the secure storing of your secrets as well as the ability to periodically and automatically rotate your secrets.
+
+This document will guide you through setting up the essential resources to access your secrets in AWS Secrets Manager and download the External Secrets Operator Helm chart. This way, you can effectively utilize your stored secrets with Helm.
+
+## Download External Secrets Operator and Create Resources in AWS.
+You can use the following Bash script to apply the External Secrets Operator to your cluster and create the necessary AWS resources. Fill in the variables below to get started:
+
+***Notice:
+The Gen3 Helm chart has various jobs and uses for an IAM user. To enhance code reusability, we've implemented the option for jobs and services to share the same AWS IAM global user. If you would like to use the same IAM user for External Secrets and jobs like ["Fence Usersync"](fence_usersync_job.md) or our "AWS ES Proxy Service", you can follow [THIS](global_IAM_helm_user.md) guide that details how to setup a Helm global user. In case you opt for a global IAM user, please comment out the "create_iam_policy" and "create_iam_user" functions at the end of the script.***
+
+```
+#!/bin/bash
+
+AWS_ACCOUNT="<Your AWS Account ID>" 
+region="<Desired AWS Region>"
+iam_policy="<Name of the Policy to Create>"
```

Comment:
Should we maybe just provide a default value for this?

Reply:
I will update this to add default values for everything but the Account ID :)